html2asketch/nodeToSketchLayers.js

```diff
@@ -235,7 +235,7 @@ export default function nodeToSketchLayers(node, options) {
     fontSize: parseInt(fontSize, 10),
     lineHeight: lineHeight !== 'normal' ? parseInt(lineHeight, 10) : undefined,
     letterSpacing: letterSpacing !== 'normal' ? parseFloat(letterSpacing) : undefined,
-    fontWeight: parseInt(fontWeight, 10),
+    fontWeight: fontWeight === 'bold' ? 700 : parseInt(fontWeight, 10),
```

Comment:
let's move it to a function e.g. `parseFontWeight(string):number`. In that function you can cover both 'normal' and 'bold' if you need to, just please leave a comment that we support 'bold' and 'normal' for Electron compatibility.

BTW How did you choose the value 700 for bold? Is it a standard?

Reply:
See this document for the standard.
https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#Common_weight_name_mapping

And, what about these codes?
```javascript
fontWeight : parseInt(fontWeight, 10) || fontWeight;
```

Reply:
Ah, thanks for explanation! I think we can keep `fontWeight === 'bold' ? 700 : parseInt(fontWeight, 10)`, but can you please move it to a separate function e.g. `parseFontWeight`?